src/base-field/base-field.tsx

```diff
@@ -7,6 +7,9 @@ import { Stack } from '../stack'
 
 import type { WithEnhancedClassName } from '../utils/common-types'
 import { Spinner } from '../spinner'
+import { Column, Columns } from '../columns'
+
+const MAX_LENGTH_THRESHOLD = 10
```

Comment:
I'd add a note in Figma for this implementation detail.

Reply:
I've shared the feedback with Julia.